Lib/test/test_import/__init__.py

```diff
@@ -775,6 +775,11 @@ def check_relative():
         ns = dict(__package__=object())
         self.assertRaises(TypeError, check_relative)
 
+    def test_import_shadowed_by_global(self):
+        # Regression test for https://bugs.python.org/issue37409
+        script_helper.assert_python_failure('-W', 'ignore', '-c',
```

Comment:
Because this `__init__.py` is expected to be loaded as a package, whereas the bugfix is about scripts, interactive sessions, etc. (So without starting a new interpreter, the relative import would be expected to always succeed, uninformatively.)

My other thought behind this test was to focus on the behaviour from a user perspective while being agnostic about all details of how `import` statements are implemented under the hood. This test would stay relevant even if the import internals were refactored (in contrast to the other test, narrowly targeting the specific `__import__` case, which might not always be applicable).